### PR TITLE
Fix deadlock when reading and writer is closed.

### PR DESCRIPTION
**Pull request by wnke**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module github.com/acomagu/bufpipe
+module github.com/wnke/bufpipe
```
Comment (acomagu) [inline go.mod]: Could you revert this?

File: bufpipe.go [hunk 2]
```diff
@@ -124,5 +124,6 @@ func (w *PipeWriter) CloseWithError(err error) error {
 		err = io.EOF
 	}
 	w.rerr = err
+	w.cond.Signal()
```
Comment (acomagu) [inline bufpipe.go]: I think this should be `Broadcast()`, don't you?

**Review by acomagu — CHANGES_REQUESTED** [review]
Nice catch! Thank you!

File: bufpipe.go (hunk 2 above)
Comment (wnke) [reply bufpipe.go]: Yes, updated!

File: go.mod (hunk 1 above)
Comment (wnke) [reply go.mod]: Of course.

**Review by acomagu — APPROVED** [review]
Thank you!!!

**Pull request merged**